docs_outdoornav_user_manual/api/api_endpoints.mdx

```diff
@@ -842,14 +1067,112 @@ string firmware_revision
 
 #### cpr_safety_msgs/WatchdogStatus {#msg-safety-watchdog}
 
-``` 
+```
 # Watchdog status message containing information related to sensor monitoring, emergency stop and safety stops
 
 bool[] gps_watchdog_triggered
 bool[] lidar_watchdog_triggered
 bool[] camera_watchdog_triggered
 ```
 
+#### onav_mission_manager_msgs/msg/Mission {#storage-mission}

```

Comment:
Just a question on the naming convent. Maybe @jmastrangelo-cpr and @civerachb-cpr have discussed this already. I see that all the messages above start with "cpr_" rather than "onav_". Is that because the previous messages are intended to be part of shared/published API with OTTO whereas the mission manager is intended to be separate from that joint activity?

Reply:
We discussed this a while ago with Steve as well. I believe the decision was to avoid using the `cpr_` prefix in our package names to avoid conflicts with Otto's packages.  I'm open to revisiting that discussion though; I have no strong opinions either way.

Reply:
Thanks. As long as it has been discussed, no need to re-open that discussion.